frontend/src/components/VExternalSearch/VExternalSearchForm.vue

```diff
@@ -139,6 +146,11 @@ export default defineComponent({
 
     const isVisible = ref(false)
 
+    // Use the `_searchType` from mediaStore because it falls back to ALL_MEDIA
```

Comment:
The comment says something different from what's happening. `searhType` is being used from the `searchStore` on [L169](https://github.com/WordPress/openverse/pull/2450/files#diff-67e95d7c4c6f5a2baf6a8450f328725599a69e6c51bc6d08546b15c4a362714dR169).